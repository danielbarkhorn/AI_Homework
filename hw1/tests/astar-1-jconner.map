 1  1  1  2  1
 1 22  1  2  1
 1  1  1 22  1
22 22 22  2  1
 1  1  5  1 22
