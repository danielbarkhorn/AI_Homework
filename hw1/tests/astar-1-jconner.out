Solution steps: ['E', 'E', 'E', 'N', 'E', 'N', 'N', 'N']
Solution cost: 30
Number of states considered: 9

Frontier:
Pos=(3, 3) Moves=['E', 'E', 'E', 'N', 'E', 'N', 'N', 'W'] Cost=31

Closed List:
Pos=(0, 0) Moves=[] Cost=0
Pos=(1, 0) Moves=['E'] Cost=1
Pos=(2, 0) Moves=['E', 'E'] Cost=18
Pos=(3, 0) Moves=['E', 'E', 'E'] Cost=23
Pos=(3, 1) Moves=['E', 'E', 'E', 'N'] Cost=25
Pos=(4, 1) Moves=['E', 'E', 'E', 'N', 'E'] Cost=27
Pos=(4, 2) Moves=['E', 'E', 'E', 'N', 'E', 'N'] Cost=28
Pos=(4, 3) Moves=['E', 'E', 'E', 'N', 'E', 'N', 'N'] Cost=29
Pos=(4, 4) Moves=['E', 'E', 'E', 'N', 'E', 'N', 'N', 'N'] Cost=30
